modules/core/src/test/resources/logback-test.xml

```diff
@@ -6,7 +6,7 @@
     </encoder>
   </appender>
 
-  <root level="ERROR" additivity="false">
+  <root level="INFO" additivity="false">
```

Comment:
This makes logs a lot noisier but I think it's more likely to be what you want if you're actually trying to track down a problem in them.